app.yml

```diff
@@ -16,7 +16,7 @@
       readiness:
         initialDelaySeconds: 15
 
-engine: ~1
+engine: 0.0.0-pr121+d2d95cb
```

Comment:
I will update when the charts pr is merged

Reply:
I'm only missing a review https://github.com/dialoguemd/charts/pull/121 then I'll put back the original value

Reply:
done

Reply:
Great, thanks!